i-slide.js

```diff
@@ -235,6 +235,7 @@ class ISlide extends HTMLElement {
       return `
         :host {
           display: block;
```

Comment:
```suggestion
          display: inline-block;
```

Reply:
I'll do that but will also rebase and update the demo and the rest of the code, given that the switch means that there needs to be a wrapping block element somewhere.